### ISSUE TITLE
Move functionality from `Artifact` class to  `ArtifactCommand` class

### ISSUE THREAD

**Issue by AlexSkrypnyk** [drevops/git-artifact#76]
#42 allows us to further improve the architecture of this package.

BLOCKED by #64

---

We now need to reconcile `Artifact` and `ArtifactCommand` classes.

This ticket is about moving the functionality of the `Artifact` to `ArtifactCommand`.

Please note that there is be a follow-up ticket to replace the logger with `monolog`.

## Solution direction.

1. Merge the functionality of `Artifact` to `ArtifactCommand`. The `Artifact::artifact()` should become `ArtifactCommand::processArtifact()` and be called from `ArtifactCommand::execute()`.

2. Update `ArtifactCommand::execute() to correctly return exit codes on exception.
The code should look like:

```php
  protected function execute(InputInterface $input, OutputInterface $output): int {
    $this->logger = self::logInit($output);

    $this->checkRequirements();

    try {
      $this->processArtifact();
    }
    catch (\Exception $e) {
      $output->writeln('<error>' . $e->getMessage() . '</error>');

      return Command::FAILURE;
    }

    return Command::SUCCESS;
  }
```
